staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4716,7 +4716,7 @@ type EphemeralContainerCommon struct {
 	// already allocated to the pod.
 	// +optional
 	Resources ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,8,opt,name=resources"`
-	// Resources resize policy for the container.
+	// Resources resize policy for ephemeral containers are not allowed.
```

Comment:
```suggestion
	// Resources resize policy for containers. ephemeral containers are not allowed.
```

something like this. I think you want to keep the description and state that ephemeral resize is not allowed.